edx_rest_framework_extensions/auth/jwt/tests/test_urls.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
```

Comment:
Can this be colocated and defined within the test file itself?    If not, can we rename this file so it's clear that it doesn't include tests itself?  The test runner will also expect this named file to contain tests.

Reply:
I would like to co-locate if possible.  I was just trying to get it to work at first.  I'll try that.

Reply:
You could also rename this file to `url_patterns.py` and keep it in the `tests` folder.

Reply:
Done.